Documentation/contributor-guide/roadmap.md

```diff
@@ -0,0 +1,47 @@
+# roadmap
+
+etcd uses GitHub milestones to track all tasks in each major or minor release. The roadmap.md file only records the most
+important tasks for each release.
```

Comment:
I would note that list is based on current maintainers capacity that will shift over time. Proposed milestones is what we think we can deliver with people we have. If we had more support on the important stuff, we could squeeze in more things from backlog.

Reply:
agreed & updated